### DISCUSSION TITLE
❓ [Question] Support for CUDA 11.6?

### DISCUSSION THREAD

**Discussion by alercelik** [pytorch/TensorRT#1258] [Q&A] [answered]
##  Does latest version support CUDA 11.6❓

Pytorch officially supports CUDA 11.6, however docs say torch_tensort supports CUDA 11.3 at max. But in some issues it is said that CUDA version 11.6 is used. Is CUDA 11.6 officially supported by torch_tensorrt?

## Environment

 - PyTorch Version (e.g., 1.0): any
 - CPU Architecture:
 - OS (e.g., Linux): Linux
 - How you installed PyTorch (`conda`, `pip`, `libtorch`, source): pip
 - Build command you used (if compiling from source):
 - Are you using local sources or building from archives:
 - Python version: 3.8 or 3.9
 - CUDA version: 11.6
 - GPU models and configuration:
 - Any other relevant information:

**Comment by narendasan** [comment]
Do you mean CUDA 11.6?

**Comment by alercelik** [comment]
Sorry, yes. CUDA 11.6

Correcting the issue for further reads.

**Comment by narendasan** [comment, marked as answer]
This will come with Torch-TensorRT 1.2.0 which targets PyTorch 1.12.0
